SHELL_AWARENESS_README.md

```diff
@@ -0,0 +1,209 @@
+# Shell Awareness Feature for Amazon Q CLI
+
+## Overview
+
+This feature adds shell awareness to Amazon Q CLI, allowing it to detect and display the correct shell-specific tool name (e.g., `execute_zsh` instead of `execute_bash`) when running commands.
+
+## The Problem
+
+The original Q CLI always showed `execute_bash` in the UI, regardless of what shell the user was actually running. This was misleading for users running zsh, fish, or other shells, as it suggested the tool wasn't aware of their shell environment.
```

Comment:
Wonder if we can get away by just changing it to `execute_shell` from `execute_bash`, usually Q won't do anything that is fish/zsh specific right?